<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QTabWidget" name="mainTab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabExpenses">
       <attribute name="title">
        <string>&amp;Expenses</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTableView" name="tableExpenses">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btnAddExpense">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabBudget">
       <attribute name="title">
        <string>&amp;Budget</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabValuables">
       <attribute name="title">
        <string>&amp;Valuables</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabSettings">
       <attribute name="title">
        <string>&amp;Settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Categories</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QToolBar" name="tbCategories"/>
           </item>
           <item>
            <widget class="QTreeView" name="treeCategories"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QGroupBox" name="groupPayers">
            <property name="title">
             <string>Payers</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QTableView" name="tablePayers"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupPayees">
            <property name="title">
             <string>Payees</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QTableView" name="tablePayees"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>This is an example application.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <widget class="QMenu" name="menu_Language">
     <property name="title">
      <string>&amp;Language</string>
     </property>
     <addaction name="actLanguageEnglish"/>
     <addaction name="actLanguageGerman"/>
    </widget>
    <addaction name="menu_Language"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Settings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actLanguageEnglish">
   <property name="text">
    <string notr="true">&amp;English</string>
   </property>
  </action>
  <action name="actLanguageGerman">
   <property name="text">
    <string notr="true">&amp;Deutsch</string>
   </property>
  </action>
  <action name="actCategoryAdd">
   <property name="text">
    <string>Add Category</string>
   </property>
  </action>
  <action name="actCategoryDelete">
   <property name="text">
    <string>Remove Category</string>
   </property>
   <property name="toolTip">
    <string>Delete Selected Category/Categories</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
